src/openapiGenerator/mockGenerator/formatResponse.ts

```diff
@@ -35,10 +35,38 @@ export const extractResponses = (obj: OpenAPI.Document): ResponsesType => {
                 } else if (response.content && response.content[APPLICATION_JSON]?.schema) {
                     mock = response.content[APPLICATION_JSON].schema;
                 }
-                mock = typeof(mock) === "string" || isExample ? mock : generateData("", mock, modelSchemas);
+                mock = typeof (mock) === "string" || isExample ? mock : generateData("", mock, modelSchemas);
                 extracted[key][statusCode] = mock;
             });
         });
     });
     return extracted;
-};
\ No newline at end of file
+};
+
+/**
+ * format path to key
+ * @param path api path
+ * @param method method name
+ * @returns formatted path
+ */
+function formatPathToKey(path: string, method: string): string {
+    let newString = "";
+    const pathArr = path.split("/").slice(1);
+    pathArr.forEach((item: string, index: number) => {
+        let newItem = item.replace(/[^\w\s]/gi, "");
+        if (index > 0) {
+            newItem = sentenceCase(newItem)
+        }
+        newString += newItem;
+    })
+    return newString + sentenceCase(method);
+}
```

Comment:
missing types for `newString`, `pathArr` and `newItem`

Reply:
added